src/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import { email, validatePassCoincidence } from 'utils/inputValidations';
+
+import wLogo from '../../assets/wLogo.png';
+
+import 'scss/layout.scss';
```

Comment:
Esto deberia estar dentro del styles.module.scss con @import 'layout'; Same for all the styles you import down here